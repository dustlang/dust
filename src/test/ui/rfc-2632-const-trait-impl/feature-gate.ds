// revisions: stock gated
// gate-test-const_trait_impl

#![cfg_attr(gated, feature(const_trait_impl))]
#![allow(incomplete_features)]
#![feature(dustc_attrs)]

struct S;
trait T {}
impl const T for S {}
//[stock]~^ ERROR const trait impls are experimental

#[dustc_error]
fn main() {} //[gated]~ ERROR fatal error triggered by #[dustc_error]
