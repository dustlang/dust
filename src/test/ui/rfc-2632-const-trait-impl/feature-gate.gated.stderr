error: fatal error triggered by #[dustc_error]
  --> $DIR/feature-gate.rs:14:1
   |
LL | fn main() {}
   | ^^^^^^^^^

error: aborting due to previous error

